package.json

```diff
@@ -256,5 +256,9 @@
     "webpack-manifest-plugin": "^3.0.0",
     "whatwg-fetch": "2.0.3",
     "xhr": "^2.5.0"
+  },
+  "devDependencies": {
```

Comment:
Don't use `devDependencies`. In a WebPack environment it's irrelevant and since we're not at all (on purpose) or consistently apply this distinction when we do then these should remain just in the dependencies list.